src/maxwell-web-view.js

```diff
@@ -27,13 +27,20 @@ let children = [];             /* List of children */
 let children_hash = new Map(); /* Hash table of children */
 
 function update_position_size () {
+    let scale = window.devicePixelRatio;
     let positions = null;
 
     for (let i = 0, len = children.length; i < len; i++) {
         let child = children[i];
         let child_rect = child.maxwell.rect;
         let rect = child.getBoundingClientRect();
 
+        /* FIXME: getBoundingClientRect() returns a wrong value when zoom is set */
+        if (scale !== 1) {
+            rect.x = (rect.x - window.scrollX * (scale - 1)) / scale;
```

Comment:
For scale = 2, `rect.x = (rect.x - window.scrollX * 1) / 2` which seems correct
For scale = 4, `rect.x = (rect.x - window.scrollX * 3) / 4` which seems incorrect.
Are you sure about the `(scale - 1)`?